crates/uv-cli/src/lib.rs

```diff
@@ -4721,6 +4721,23 @@ pub enum PythonCommand {
     /// See `uv help python` to view supported request formats.
     Install(PythonInstallArgs),
 
+    /// Upgrade installed Python versions to the latest supported patch release.
+    ///
+    /// A target Python minor version to upgrade may be provided, e.g., `3.13`. Multiple versions
+    /// may be provided to perform more than one upgrade.
+    ///
+    /// If no target version is provided, then uv will upgrade all managed CPython versions.
+    ///
+    /// Upgrades will not remove lower installed patch versions.
```

Comment:
I think this is kind of confusing as written

```suggestion
    /// During an upgrade, uv will not uninstall outdated patch versions.
```